### DISCUSSION TITLE
openAI response formatting

### DISCUSSION THREAD

**Discussion by Ahzam-Hassan** [gonzalezreal/swift-markdown-ui#375] [Q&A]
Has anyone tried to format the response of openAI using this library? i am trying to but the math equations are not formatted properly

**Comment by rafaelpedretti-toast** [comment]
Look if https://github.com/gonzalezreal/swift-markdown-ui/discussions/208 helps you

**Reply by Ahzam-Hassan** [reply]
Thanks for your response, i will look into it
